test/builder/builder_test.cc

```diff
@@ -275,6 +276,49 @@ TEST_F(RoadGeometryBuilderTest, LaneGeometryBuilderWithMissingLineStrings) {
       maliput::common::assertion_error);
 }
 
+TEST_F(RoadGeometryBuilderTest, LaneGeometryBuilderWithAndWithoutCenter) {
+  // No throws as LaneGeometryBuilder is properly used.
+  // Centerline isn't passed to the builder.
+  EXPECT_NO_THROW(
+      // clang-format off
+      RoadGeometryBuilder()
+          .StartJunction()
+              .Id(kJunctionAId)
+              .StartSegment()
+                  .Id(kSegmentAId)
+                  .StartLane()
+                      .Id(kLaneAId)
+                      .StartLaneGeometry()
+                          .LeftLineString(kLeftLineStringA)
+                          .RightLineString(kRightLineStringA)
+                      .EndLaneGeometry()
+                  .EndLane()
+              .EndSegment()
+      // clang-format on
+  );
+
+  // No throws as LaneGeometryBuilder is properly used.
+  // Centerline is submitted to the builder.
+  EXPECT_NO_THROW(const auto kCenterline{geometry::utility::ComputeCenterline3d(kLeftLineStringA, kRightLineStringA)};
+                  // clang-format off
```

Comment:
I think this is an odd column spacing, isn't it? Same comment below.

Reply:
Done.